docs/systemd.md

```diff
@@ -168,3 +168,24 @@ Apr 07 08:40:19 hx puma[28320]: * Activated tcp://0.0.0.0:9233
 Apr 07 08:40:19 hx puma[28320]: * Activated ssl://0.0.0.0:9234?key=key.pem&cert=cert.pem
 Apr 07 08:40:19 hx puma[28320]: Use Ctrl-C to stop
 ~~~~
+
+## Alternative background process configuration
+
+If Capistrano and [capistrano3-puma](https://github.com/seuros/capistrano-puma) tasks are used you can use the following configuration. In this case, you would skip systemd Socket Activation, since Puma handles the socket by itself:
+
+~~~~
+[Service]
+# Background process configuration (use with --daemon in ExecStart)
+Type=forking
+
+# Use `cap <stage> puma:start -n` in order to spot the exact command,
+# which could change for example is a Ruby version manager is used
```

Comment:
Hi!

This text about how to define a systemd Service Unit file can be improved to avoid jargon.

```
+# Use `cap <stage> puma:start -n` in order to spot the exact command,
+# which could change for example is a Ruby version manager is used
```

After reading [a little about the systemd way of defining Services](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Unit_Files.html#sect-Managing_Services_with_systemd-Unit_File_Create) and Capistrano, I propose this different wording of the example text:  

```
# To learn which exact command is to be used to execute at "ExecStart" of this
# Service, ask Capistrano: `cap <stage> puma:start --dry-run`. Your result
# may differ from this example, for example if you use a Ruby version
# manager. `<WD>` is short for "your working directory". Replace it with your
# path.
```